js-task-second/js/main.js

```diff
@@ -0,0 +1,82 @@
+const loader = document.getElementById('loader');
+const block = document.getElementById('block');
+const title = document.getElementById('title');
+const listOfArticles = [];
+
+async function requestOfDate () {
+  const response = await fetch('https://api.punkapi.com/v2/beers/random', {
+    method: "GET",
+    mode: "cors",
+    cache: "no-cache",
+    credentials: "same-origin",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    redirect: "follow",
+    referrerPolicy: "no-referrer",
+  });
+  return response.json();
+}
+
+function onClick (event) {
+  const target = event.currentTarget;
+  const lastElement = target.children.length - 1;
+  if (target.children[lastElement].style.display === 'flex') {
+    target.children[lastElement].style.display = 'none';
+    return;
+  }
+  target.children[lastElement].style.display = 'flex';
+}
+
+window.addEventListener('load', async () => {
+  for (let step = 0; step < 10; step++) {
+    const article = await requestOfDate();
+    listOfArticles.push(article);
```

Comment:
in the task it was mentioned that we need to make 10 requests at once (to practice Promise.all) but in your case you make the requests one by one